src/GitHub.UI/Converters/BranchNameConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using GitHub.Models;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Data;
+
+namespace GitHub.UI.Converters
+{
+    public class BranchNameConverter : IMultiValueConverter
+    {
+
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
+        {
+            var branch = values[0] as IBranch;
+            var activeRepo = values[1] as IRepositoryModel;
```

Comment:
might not something like this be better?
```c#
var branch = values.OfType<IBranch>().FirstOrDefault();
var activeRepo = values.OfType<IRepositoryModel>().FirstOrDefault();
```
will automatically work for when you fail to provide all values and will even work for when you provide the parts in a different order.